dcc-portal-ui/app/scripts/pathwayviewer/js/pathways.js

```diff
@@ -60,42 +60,42 @@
                                                     GeneSetService, GeneSetHierarchy, GeneSets,
                                                     GeneSetVerificationService, TooltipText,
                                                     LocationService, EnrichmentService, SetService,
-                                                    PathwaysConstants, RestangularNoCache) {
-    
-    
+                                                    PathwaysConstants, RestangularNoCache, PathwayDataService) {
+
+
     var _selectedPathway = null;
-    
+
     function _resolveEnrichmentData(Id) {
       $scope.analysis.isLoading = true;
-      
+
       SetService.pollingResolveSetFactory(Id, 2000, 10)
-        .setRetrievalEntityFunction(function () {
-          return RestangularNoCache.one('analysis/enrichment', Id).get();
-        })
+      	.setRetrievalEntityFunction(function () {
+        	return RestangularNoCache.one('analysis/enrichment', Id).get();
+      	})
         .setResolvedEntityFunction(function (entityData) {
           return entityData.state.toUpperCase() === 'FINISHED';
         })
         .resolve()
-        .then(function (entityData) {
-          
-          _initAnalysisScope(entityData);
-          
-          if (!_selectedPathway) {
+      	.then(function (entityData) {
+
+	        _initAnalysisScope(entityData);
+
+        	if (!_selectedPathway) {
             return;
-          }
-          
-          var _updateSelectedPathway = _.first(
-            _.filter(entityData.results, function (pathway) {
-              return pathway.geneSetId === _selectedPathway.geneSetId;
-            })
-          );
-          
-          _.assign(_selectedPathway, _updateSelectedPathway);
-          $scope.analysis.isLoading = false;
-          
-        });
+        	}
+
+       	var _updateSelectedPathway = _.first(
```

Comment:
Please also turn on whitespace visibility. I know it'll look weird at first but trust me it only takes an hour to get used to